src/test/regress/expected/pg14.out

```diff
@@ -1376,6 +1376,35 @@ SELECT create_distributed_table('ctlt1', 'a');
 (1 row)
 
 CREATE TABLE ctlt_all_2 (LIKE ctlt1 INCLUDING ALL);
+CREATE TABLE compression_and_defaults (
+    data text COMPRESSION lz4 DEFAULT '"{}"'::text COLLATE "de_DE" NOT NULL PRIMARY KEY,
+    rev text
+)
+WITH (
+    autovacuum_vacuum_scale_factor='0.01',
+    fillfactor='75'
+);
```

Comment:
We see some errors on CI here.

    ERROR:  collation "de_DE" for encoding "UTF8" does not exist

Maybe we should find some other collation that is available on CI images. Updating the CI images is also another option, but it will not be a trivial fix.

Reply:
I usually test with "C"